src/components/Activity/DefaultTabBar.js

```diff
@@ -1,33 +1,41 @@
-const React = require('react');
-const ReactNative = require('react-native');
-const {
+import React, { Component } from 'react';
+import {
+  Animated,
   Dimensions,
   StyleSheet,
-  Text,
   View,
-} = ReactNative;
-const Button = require('./Button');
+ } from 'react-native';
+import Button from './Button';
+import Text from '../Shared/UniText';
 
 class DefaultTabBar extends Component {
+  static get defaultProps() {
+    return {
+      activeTextColor: '#2e3031',
+      inactiveTextColor: '#adadad',
+      backgroundColor: null,
+    };
+  }
+
   renderTab(name, page, isTabActive, onPressHandler) {
-    const { activeTextColor, inactiveTextColor, textStyle, } = this.props;
+    const { activeTextColor, inactiveTextColor, textStyle, } = DefaultTabBar.defaultProps;
```

Comment:
ESLint says only IE8 and below browser occurs error with this, So isn't it good to use trailing comma in React-Native?